Examples/E3 - Cargo/AirFactory.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Examples.Cargo;
+
+public class AirFactory : ITransportFactory
```

Comment:
you have factory , so why you have not managed you air instance here?

Reply:
I think the factory should only be concerned with creating the air transport, not managing its state. Is this incorrect? Should I manage the creation of a single instance within the factory instead?
@mohammadKarimi